commons/src/main/java/gov/cms/qpp/conversion/model/error/FormattedErrorCode.java

```diff
@@ -34,13 +34,17 @@ public String toString() {
 
 	@Override
 	public boolean equals(Object o) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1142)